### DISCUSSION TITLE
Support use 2 wireless cards ?

### DISCUSSION THREAD

**Discussion by ghost** [ZerBea/hcxdumptool#428] [General]
hcxdumptool -i wlan0  -i wlan1  -F 
If one for monitoring and other for attack,  there is no need to stay in the channel for 6s.
When using 2 wireless cards, specify a wireless card loops all channel AP attack. (each channel only needs attack for 1s)
This will save time and reduce waiting for attack cycles, especially in more channel situations, Wifi5 and Wifi6.
Because there is a card monitoring all channels at any time. 
It will not due switching to other channel, missed  a handshake message

**Comment by ZerBea** [comment]
I've tried this (hcxlabtool running 2, 3 or 4 interfaces):
```
ef265f8 4 interfaces on a single RPI don't work
e7e458b a RPI can't handle 4 WiFI interfaces
b5df689 a RPI can't handle 4 WiFI interfaces
```
And I gave it up:
```
heavy interferences
Raspberry Pi driver can't handle two channels simultaneous in monitor mode.
```

It may work on a passive dumper, but it will not work on hcxdumptool.
